lib/trackler/problems.rb

```diff
@@ -9,8 +9,8 @@ def initialize(root)
     end
 
     def each
-      active.each do |problem|
-        yield problem
+      active.each do |specification|
+        yield specification
       end
```

Comment:
Commit level comment: 

At this point we have a `Problems` class that only references `specifications` and never `problems`.
If we were to move the code into the `Specifications` class and reverse the dependency would that change the API at all?